### PR TITLE
Example: Uid filtering

### PR DESCRIPTION
**Pull request by mzidek-gh**
Add example of how to filter crashes based on Uid.

Resolves:
https://github.com/abrt/abrt/issues/1226

### CODE REVIEW

File: examples.rst [hunk 1]
```diff
@@ -270,3 +270,30 @@ Set the new `kernel.core_pattern` using `sysctl` (basically, change
...
 
 .. [#corepattern] https://www.kernel.org/doc/html/latest/admin-guide/sysctl/kernel.html#core-pattern
+
+
+Is it possible to ignore crashes from programs with specific UIDs?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Yes, it is. You can modify files in `/etc/libreport/events.d/` and add
+your own snippets of bash code to them. In event `post-create` (see for example
+`etc/libreport/events.d/python3_event.conf` for Pythin 3 crashes or
```
Comment (xsuchy) [inline examples.rst:280]: Pyth**o**n

File: examples.rst [hunk 2]
```diff
@@ -270,3 +270,30 @@ Set the new `kernel.core_pattern` using `sysctl` (basically, change
...
+`etc/libreport/events.d/ccpp_event.conf` for C and C++ crashes) you
+have access to contents of /proc/PID/status file of the process that
```
Comment (xsuchy) [inline examples.rst:282]: backticks

File: examples.rst [hunk 3]
```diff
@@ -270,3 +270,30 @@ Set the new `kernel.core_pattern` using `sysctl` (basically, change
...
+crashed.
+
+Example how to filter crashes from programs with UIDs lower than 1000 in C
+and C++ programs:
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Open  `etc/libreport/events.d/ccpp_event.conf` and add this snippet in
+the EVENT=post_create section:
```
Comment (xsuchy) [inline examples.rst:290]: backticks

File: examples.rst [hunk 4]
```diff
@@ -270,3 +270,30 @@ Set the new `kernel.core_pattern` using `sysctl` (basically, change
...
+.. code:: bash
+
+        # Parse Uid from  proc_pid_status
+        uid=`grep '^Uid:' proc_pid_status | sed 's/^Uid:[[:space:]]*\([1234567890]*\).*/\1/'`
```
Comment (xsuchy) [inline examples.rst:294]: The numbers can be shorten to [0-9]

File: examples.rst [hunk 5]
```diff
@@ -270,3 +270,30 @@ Set the new `kernel.core_pattern` using `sysctl` (basically, change
...
+        
+        if [ 1000 -lt $uid ]; then
```
Comment (xsuchy) [inline examples.rst:296]: uid should be quoted

**Comment by xsuchy** [comment]
beside these nitpicks it looks good.

**Comment by mzidek-gh** [comment]
Thanks for the review. Patch updated.

**Review by xsuchy — APPROVED** [review]

File: examples.rst [hunk 6]
```diff
@@ -270,3 +270,30 @@ Set the new `kernel.core_pattern` using `sysctl` (basically, change
...
+`etc/libreport/events.d/python3_event.conf` for Python 3 crashes or
+`etc/libreport/events.d/ccpp_event.conf` for C and C++ crashes) you
+have access to contents of `/proc/PID/status` file of the process that
+crashed.
+
+Example how to filter crashes from programs with UIDs lower than 1000 in C
+and C++ programs:
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Open  `etc/libreport/events.d/ccpp_event.conf` and add this snippet in
```
Comment (mgrabovsky) [inline examples.rst:289]: Missing the initial slash before `etc` (and two times above as well).

**Comment by mzidek-gh** [comment]
My eyes are parsing weirdly today. Thanks for the catches. New patch uploaded.

**Comment by mzidek-gh** [comment]
W8 please, I think I have broken the formatting of the code section :)

**Comment by mzidek-gh** [comment]
Now it is correct.

**Review by mgrabovsky — APPROVED** [review]
Looks good to me now, as well.

**Pull request merged**